nipoppy/workflows/dataset_init.py

```diff
@@ -146,49 +146,54 @@ def _init_manifest_from_bids_dataset(self) -> None:
 
         self.logger.info("Creating a manifest file from the BIDS dataset content.")
 
-        for ppt in participant_ids:
+        for bids_participant_id in bids_participant_ids:
 
-            session_ids = sorted(
+            bids_session_ids = sorted(
                 [
                     x.name
-                    for x in (self.layout.dpath_bids / ppt).iterdir()
+                    for x in (self.layout.dpath_bids / bids_participant_id).iterdir()
                     if x.is_dir() and x.name.startswith(BIDS_SESSION_PREFIX)
                 ]
             )
-            if len(session_ids) == 0:
+            if len(bids_session_ids) == 0:
                 # if there are no session folders
                 # we will add a fake session for this participant
-                session_ids = [FAKE_SESSION]
+                self.logger.warning(
+                    "Could not find session-level folder(s) for participant "
+                    f"{bids_participant_id}, using {FAKE_SESSION_ID} in the manifest"
```

Comment:
```suggestion
                    f"{bids_participant_id}, using session {FAKE_SESSION_ID} in the manifest"
```

Since `FAKE_SESSION_ID` has no BIDS prefix now